edunext_openedx_extensions/ednx_microsites/backends/database.py

```diff
@@ -12,7 +12,7 @@
     from microsite_configuration.backends.base import BaseMicrositeBackend
 except ImportError:
     from .base import BaseMicrositeBackend
-from microsite_configuration.models import Microsite
+from ..models import Microsite
```

Comment:
I was using relative imports only for module dependencies inside the same app (in this case ednx_microsites). There are another relative imports in edunext, microsite_api and manage_api app. Do you agree with doing this change across all EOE in this PR? or better create other PR, and then rebase this one?